### ISSUE TITLE
Carom3D Server IP

### ISSUE THREAD

**Issue by evandropoa** [cgreche/open-carom3d-server#2]
It´s not need to mess with windows HOST file, just create a folder and a file inside carom3d installed folder like this **ServerAddress/ServerAddress.txt**

Then list the server or servers ips or hostname in the text file like this:
10.0.0.1
127.0.0.1
localhost

**Comment by cgreche** [comment]
You are right. I discovered that recently.
But for custom profiles without carom.exe modification, it's needed to set a hosts entry as follow:
SERVER_EXPOSED_IP carom3d.gameis.com

By the way, internal HTTP server coming to 0.3.0-ALPHA version :)

**Comment by evandropoa** [comment]
Yep, there´s so many references to carom3d.gameis.com, but it´s there to support profile http aplication integration :)

**Comment by felipeaav** [comment]
it does not work on an internal network? I even managed to connect to the server, but after that it doesn't start the room. Would it work with no-ip? how would i configure the router and ports?
